include/nbl/builtin/glsl/ext/Scan/scan.glsl

```diff
@@ -84,7 +83,7 @@ barrier();\
 uint data = global_offset;\
 if (gl_LocalInvocationIndex != 0u && (gl_GlobalInvocationID.x < nbl_glsl_ext_Scan_Parameters_t_getElementCountPass()))\
 {\
-	uint prev_idx = STRIDED_IDX(gl_GlobalInvocationID.x - 1u);\
+	uint prev_idx = gl_GlobalInvocationID.x * nbl_glsl_ext_Scan_Parameters_t_getStride() - 1u;\
```

Comment:
are you sure this does the same thing as before?

Reply:
yep

actually, `((gl_GlobalInvocationID.x - 1u) + 1u) * nbl_glsl_ext_Scan_Parameters_t_getStride() - 1u`

`gl_GlobalInvocationID.x - 1u` because this strided index corresponds to the previous thread